service/src/main/java/gov/nasa/pds/api/registry/model/PdsLid.java

```diff
@@ -0,0 +1,36 @@
+package gov.nasa.pds.api.registry.model;
```

Comment:
Kind of surprised we don't already have a "data class" like this, perhaps in `pds4-jparser`. Oh well, better late than never!

Reply:
We may do, but if so it's not in use in _this_ repo.  Is it worth me following that up, or am I good to continue with this for this repo, at least?

Reply:
You know what?  After looking at it I think that module might be _slight_ overkill for the task of sorting VIDs (actually I don't even think it's capable of that, but it *is* very imposing and impressive)